introduction/principles.md

```diff
@@ -18,11 +18,11 @@ Let's take a look at each of these principles in more detail.
 
 ## Openness
 
-In being open, a project is set up in such a way that contributing is as frictionless as possible.
-Projects should be discoverable and well documented.
-Anyone within an organization should be able to find them and ramp up without an inordinate amount of direct guidance from host team members.
+The configuration of an open project enables frictionless contributing.
+Projects should be discoverable and well documented through README.md and CONTRIBUTING.md files within the root of the repo.
+Anyone within an organization should be able to find a desired project and ramp up without an inordinate amount of direct guidance from host team members.
 Host team contact information should be prevalent with as many channels as makes sense for the project.
-Host team intentions to InnerSource their project should be broadcast through relevant organization channels.
+Host team intentions to InnerSource their project can be broadcast through relevant organization channels to raise awareness. 
```

Comment:
Do we want to make any recommendation on marketing things here?

Reply:
Done. Pls. let me know what you think.